src/custom-types.json

```diff
@@ -501,6 +501,13 @@
   "application/zip": {
     "compressible": false
   },
+  "application/dicom": {
+    "source": [
+        "https://en.wikipedia.org/wiki/DICOM#"
```

Comment:
Hello, and thank you. The contribution guide should be clear that links to Wikipedia are not acceptable. Please link to the primary source, i.e. where that Wikipedia sourced the information for this.

Reply:
I forgot to add that I see you also put `compressible: true`, so please make sure there is also at least one source included for how the determination of compressible was determined.